src/mapgen.cpp

```diff
@@ -10099,13 +10099,12 @@ FFFFFFFFFFFFFFFFFFFFFFFF\n\
         }
     }
 
-    int terrain_type_with_suffix_to_nesw_array( oter_id terrain_type, bool array[4] );
```

Comment:
Yuck, this isn't your fault or anything, but it's terrible.  If you have a moment please move this prototype to a header file and include it instead of declaring it here.